cfgov/unprocessed/js/modules/util/Behavior.js

```diff
@@ -0,0 +1,34 @@
+/* ==========================================================================
+   Get Breakpoint State
+   ========================================================================== */
+
+'use strict';
+
+var Behavior = {};
+
+/**
+ * @param {string} selector Used to query dom for elements.
+ * @param {string} event Event type to add to element.
+ * @param {function} eventHandler Callback for event.
+ */
+Behavior.create = function create( selector, event, eventHandler ) {
```

Comment:
I've modified the code to allow the user to pass in a Node, NodeList or a selector. I've also moved the find function from the [hook utility](https://github.com/cfpb/cfgov-refresh/blob/3a3b6434ee9c39354b6db37235710f32fe261cca/cfgov/unprocessed/js/modules/util/data-hook.js#L68) to the behavior module. Let me know what you think about this approach.